src/app/components/Home/ListOfAddresses/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import List from "@material-ui/core/List";
+import ListItem, { ListItemProps } from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Divider from "@material-ui/core/Divider";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import { ListItemIcon } from "@material-ui/core";
+
+const ListItemLink = (props: ListItemProps<"a", { button?: true }>) => {
+  return (
+    <>
+      <li>
+        <ListItem button component="a" {...props} />
+      </li>
+      <Divider variant="middle" />
+    </>
+  );
+};
+
+interface IarrCity {
+  [key: number]: string;
+}
+
+interface IListOfAddresses {
+  addresesData: Array<IarrCity>;
+}
+
+const ListOfAddresses: React.FC<IListOfAddresses> = ({ addresesData }) => {
+  return (
+    <>
+      <List component="nav" aria-label="secondary mailbox folders">
+        {addresesData?.map((keyName, item, index) => (
```

Comment:
Callback inside `map` gets `item`, `index` and `array` as arguments. In your case `keyName` is address, `item` is index of item in array and `index` is an array of addresses. If you use `Object.entries()` you'll get an array of pairs `[address, office_id]` and your callback inside `map` will look something like this `([address, office_id]) => {...}` and `office_id` also can be used as a key.